marklogic-client-api/src/main/java/com/marklogic/client/impl/PlanBuilderSubImpl.java

```diff
@@ -482,6 +482,13 @@ public ServerExpression seq(ServerExpression... expr) {
     return new BaseTypeImpl.ServerExpressionListImpl(expr, true);
   }
 
+  @Override
```

Comment:
Same thing as in the interface - I think this should be grouped with the other `from` methods, so I'd put this at line 165 after `fromView` and before `avg`.